operational/vmware/instance_tag_sync/instance_tag_sync.pt

```diff
@@ -138,7 +138,7 @@ script "js_instances_and_tags_and_cloud", type: "javascript" do
   parameters "ds_instances_and_tags", "ds_clouds"
   result "results"
   code <<-EOF
-    var results = []  
+    var results = []
```

Comment:
Can you please correct the line indentation

Reply:
made the change